gentest/fixtures/YGAbsolutePositionTest.html

```diff
@@ -24,3 +24,39 @@
   <div style="position: absolute; width: 50px; height: 50px; left: 0px; top: 0px;"></div>
   <div style="position: absolute; width: 50px; height: 50px; right: 0px; bottom: 0px;"></div>
 </div>
+
+<div id="absolute_layout_align_items_and_justify_content_center" style="height: 100px; width: 110px; flex-grow: 1; align-items: center; justify-content: center;">
+	<div style="position: absolute; width: 60px; height: 40px;"></div>
+</div>
+
+<div id="absolute_layout_align_items_and_justify_content_flex_end" style="height: 100px; width: 110px; flex-grow: 1; align-items: flex-end; justify-content: flex-end;">
+	<div style="position: absolute; width: 60px; height: 40px;"></div>
+</div>
+
+<div id="absolute_layout_justify_content_center" style="height: 100px; width: 110px; flex-grow: 1; justify-content: center;">
+	<div style="position: absolute; width: 60px; height: 40px;"></div>
+</div>
+
+<div id="absolute_layout_align_items_center" style="height: 100px; width: 110px; flex-grow: 1; align-items: center;">
+	<div style="position: absolute; width: 60px; height: 40px;"></div>
+</div>
+
+<div id="absolute_layout_align_items_center_on_child_only" style="height: 100px; width: 110px; flex-grow: 1;">
```

Comment:
I actually think we need this test, to keep verifing that we call ```YGNodeAlignItem``` and we won't check only on the parent node style. On lines 1396 and 1400.